localstack/services/install.py

```diff
@@ -47,6 +47,13 @@ def install_elasticsearch():
             mkdir(dir_path)
             chmod_r(dir_path, 0o777)
 
+        # install default plugins
+        for plugin in ELASTICSEARCH_PLUGIN_LIST:
+            if is_alpine():
+                # https://github.com/pires/docker-elasticsearch/issues/56
+                os.environ['ES_TMPDIR '] = '/tmp'
+            run('%s/bin/elasticsearch-plugin install %s' % (INSTALL_DIR_ES, plugin))
```

Comment:
For cross-platform compatibility, perhaps better:
```
run('%s install %s' % (os.path.join('bin', 'elasticsearch-plugin', INSTALL_DIR_ES), plugin))
```

Reply:
fixed!